src/aspire/basis/basis.py

```diff
@@ -104,14 +135,14 @@ def mat_evaluate(self, V):
             -`self.sz` corresponding to the evaluation of `V` in
             this basis.
         """
-        return mdim_mat_fun_conj(V, 1, len(self.sz), self.evaluate)
+        return mdim_mat_fun_conj(V, 1, len(self.sz), self._evaluate)
 
     def mat_evaluate_t(self, X):
         """
         Evaluate coefficient matrix in dual basis
 
-        :param X: The coefficient array of size `self.sz`-by-`self.sz`
-            to be evaluated.
+        :param X: The coefficient array as an Image or Volume object of
```

Comment:
This is still a coefficient array, no?

Reply:
Yes good point thank you. Will remove Image or Volume comment